kolena/_api/v1/generic.py

```diff
@@ -179,6 +180,13 @@ class UploadEmbeddingsRequest(BatchedLoad.WithLoadUUID):
     class UploadEmbeddingsResponse:
         n_samples: int
 
+    @dataclass(frozen=True)
+    class GetEmbeddingsModelResponse:
+        model: str
+        model_url: str  # presigned url for model
```

Comment:
Extra: following my last comment, have you checked if we can include hash of these files for client side verification (more sound check that local file cache is the same as target file)?

Reply:
Will look into this